satellite/console/consoleweb/consoleql/project.go

```diff
@@ -51,6 +55,14 @@ const (
 	FieldCurrentPage = "currentPage"
 	// FieldTotalCount is a field name for bucket usage count total
 	FieldTotalCount = "totalCount"
+	// FieldCardBrand is a field name for credit card bard
+	FieldCardBrand = "brand"
+	// FieldCardLastFour is a field name for credit card last four digits
+	FieldCardLastFour = "lastFour"
+	// FieldCardToken is a field name for credit card token
+	FieldCardToken = "cardToken"
```

Comment:
Isnt that called CVC?

Reply:
Which one?
Card Token? `CardToken` is used with stripe-elements to add payment method.
We use their API and form to add payment method. All validation and other checks are on their side, and in success case we receive card token which later will be process on our backend

Reply:
> Isnt that called CVC?

We don't store sensitive user data on our side

Reply:
Okay cool, reminded me on that one